packages/account/src/Components/api-token/_tests_/api-token-article.spec.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { screen, render } from '@testing-library/react';
+import ApiTokenArticle from '../api-token-article';
+
+describe('<ApiTokenArticle />', () => {
```

Comment:
You don't need a describe here because there is only a single test written.
Ref: https://jestjs.io/docs/api#describename-fn